Tools/peg_generator/mypy.ini

```diff
@@ -4,7 +4,7 @@ pretty = True
 show_traceback = True
 
 # Make sure the peg_generator can be run using Python 3.10:
```

Comment:
This comment needs to be updated.

(Is it no longer a goal to be able to run the peg_generator code using Python 3.10? Is it okay to require Python 3.12+ in order to build the CPython `main` branch? Should this be discussed more widely first?)

Reply:
Hummmm, I can change the PR to (optionally expect those tokens so we remain compatible with 3.10+).

Reply:
Done